integrated_channels/sap_success_factors/exporters/content_metadata.py

```diff
@@ -175,6 +196,37 @@ def transform_courserun_title(self, content_metadata_item):
 
         return title_with_locales
 
+    def _get_course_run_start_end_str(self, course_run):
+        """
+        Get the course run start and end as a descriptive string. Also include a note if enrollment is closed.
+        """
+        course_run_start = course_run.get('start')
+        course_run_end = course_run.get('end')
+        date_str = ''
+
+        if course_run_start:
+            date_str += '{starts}: {:%B %Y}'.format(
+                parse_lms_api_datetime(course_run_start),
+                starts=_('Starts')
+            )
+
+        if course_run_end:
+            if date_str:
+                date_str += ', '
```

Comment:
nit: I find string templates easier to follow than appending to strings. Feel free to ignore if you don't feel the same. Something like:
```
date_str = "{}{}{}{}".format(
    start_str,
    ", " if start_str and end_str else "",
    end_str,
    ". " if start_str or end_str else "",
)
```

Reply:
That makes sense. I think I'm going to leave this PR as-is, but I'll keep that in mind for the future.

Reply:
Sounds good!